src/services/middlewareServices/FormsProcessingService.ts

```diff
@@ -47,12 +38,12 @@ export default class FormsProcessingService {
     }
   }
 
-  decrypt = ({ formKey }: { formKey: string }) => (
+  decrypt = ({ formKey }: { formKey: string }) => async (
     req: Request,
     res: Response,
     next: NextFunction
-  ): void => {
-    const submission = this.formsg.crypto.decrypt(
+  ): Promise<void> => {
+    const submission = await this.formsg.crypto.decryptWithAttachments(
```

Comment:
Do existing forms (without attachments) work with this? do we need a conditional on attachment existence?

Reply:
yes, checked already

Reply:
Wait to be clear this was a breaking change to the api, so the other forms had to change and I checked that the other forms were able to parse the content (ie the individual fields)